README.md

```diff
@@ -52,4 +52,4 @@ Notes:
 - test deployment uses the same alza account as production - do not add stuff to cart
 - you can view existing deployments with `now ls eshop-api`
 - remove old deployments with `now remove [deployment id]` (just be careful not to delete the production deployment if you don't mean to do so).
-- [Now CLI reference](https://eshop-api-yxdgefsdby.now.sh) 
```

Comment:
Fixup this on top of the previous commit. No need to have this as two commits.

Reply:
What does "on top of the previous commit" mean please? Ammending the previous commit? Does it matter, if I will squash before merging to master?

Reply:
You should rebase. Having proper commits helps understanding and reviewing for a reviewer.